geometry_msgs/Point point  # x,y of seleceted marker in image coordinates
---
bool success  # bool feedback
